config/locales/server.en.yml

```diff
@@ -322,6 +322,7 @@ en:
         You must be signed in to proceed.
         Please <a href="/dashboard" target="_blank">sign in</a> then reload this page to continue.
       recaptcha: "We weren't able to verify that you're not a robot. Please try again!"
+      forbidden_name: "Forbidden name, please choose another name"
```

Comment:
How about "You name contains a reserved word, please remove it"

Reply:
Now I understand this a little better. "The name you have provided is a reserved keyword in Loomio".